build.sh

```diff
@@ -7,7 +7,7 @@ TEMP_PROJECT_VERSION="temp-version"
 
 # `tasks` triggers full tasks graph resolving, checking for possible misconfigurations
 runInBuilder "./gradlew tasks build publishToMavenLocal ${GRADLE_ARGS} -PprojectVersion=${TEMP_PROJECT_VERSION};
-    ./gradlew :subprojects:android-test:test-app:instrumentationUi ${GRADLE_ARGS} -PinfraVersion=${TEMP_PROJECT_VERSION}"
+    ./gradlew :subprojects:android-test:test-app:instrumentationUi ${GRADLE_ARGS} -PinfraVersion=${TEMP_PROJECT_VERSION};
+    ./gradlew help"
```

Comment:
run without GRADLE_ARGS it the same as run new docker without env

Reply:
What about running it as the first commant?

```
runInBuilder " 
1 mavenLocal
2 help
3 instrumentationUI
"

It shows problem in configuration. It's better to catch them as early as possible, in the simplest run.